test/test_datapipe.py

```diff
@@ -1220,7 +1220,7 @@ def _helper(ref_fn, fn, input_col=None, output_col=None):
                  {"x": 6, "y": 7, "z": 8}]
             )
             res_dp = datapipe.map(fn, input_col, output_col)
-            ref_dp = datapipe.map(ref_fn)
+            ref_dp = datapipe.map(ref_fn) if ref_fn is not None else datapipe
```

Comment:
Could you please also add two tests here?
- A lambda function
- A `functools.partial` function

Reply:
Like `ref_dp = datapipe.map(lambda f: fn(f), input_col, output_col)`? Or are they supposed to be new parameters to `_helper`

Reply:
Just for `test_map_dict_with_col_iterdatapipe`? Or `test_map_tuple_list_with_col_iterdatapipe` and `test_map_iterdatapipe` as well (the tests above this)

Reply:
Okay, working on it